packages/adminrouter/extra/src/docs/api/nginx.master.yaml

```diff
@@ -264,6 +272,13 @@ routes:
     description: List the active Pkgpanda packages
     file: /opt/mesosphere/active.buildinfo.full.json
     path: /pkgpanda/active.buildinfo.full.json
+  /service/(?<service_path>.+):
+    group: Service
+    matcher: regex
+    description: Proxy to services running on DC/OS
+    proxy:
+      path: $upstream_url
+    path: /service/(?<service_path>.+)
   '/service/(?<serviceid>[0-9a-zA-Z-.]+)':
```

Comment:
It's not - this entry describes a redirect location. Check the description below this line.